app/services/web_mentions/web_mention_handler.rb

```diff
@@ -0,0 +1,37 @@
+module WebMentions
+  class WebMentionHandler
+    def initialize(canonical_url, article_url = nil)
+      @canonical_url = canonical_url
+      @article_url = article_url
+      @webmention_url = ""
+    end
+
+    def self.call(*args)
+      new(*args).call
+    end
+
+    def call
+      accepts_webmention? ? send_webmention(@webmention_url) : Rails.logger.info("#{@canonical_url} doesn't support Webmentions")
```

Comment:
What is the goal fo all of the error logging? Are these failures we want someone to address?

Reply:
@mstruve I added these for debugging purposes but i don't think there's more one can do when a URL doesn't support webmentions! 

Should i remove it?